migrations/0002-changelog.sql

```diff
@@ -7,5 +7,6 @@ CREATE TABLE IF NOT EXISTS `animal_keeping`.`changelog` (
   `created_time` DATETIME NOT NULL,
   `target_type` VARCHAR(200) NOT NULL,
   `target_id` INT NOT NULL, 
+  `who` VARCHAR(200) NOT NULL,
```

Comment:
This change needs to go into a separate migration script. When we patch old migrations we will not be able to simply go on from an existing point (e.g. migration 0004 has already been applied) 
Just add a new script 0005-addusertochangelog_or_whatever_name.sql that contains the change:
````
ALTER TABLE ` animal_keeping`.`changelog` ADD `who` VARCHAR(200) NOT NULL;
```